cache/models/samples.js

```diff
@@ -466,17 +467,16 @@ function upsertOneSample(sampleQueryBodyObj, isBulk, userName) {
     // add aspect name to subject set, add sample key to sample set,
     // create/update hash of sample
     return redisClient.batch([
-      ['sadd', subjKey, aspectName],
+      ['sadd', constants.indexKey.aspect, aspectName],
```

Comment:
good point. I will take out that line.
we add aspect to subject hash with hmset instead of sadd